Validator/Validator.csproj

```diff
@@ -21,6 +21,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
```

Comment:
@ledbutter, it looks like <Prefer32Bit>false</Prefer32Bit> is what was added. I did not explicitly add this. I can safely remove it from the project and make that part of my next PR.